File: sandbox_workspace/src/hello.py
#!/usr/bin/env python3
print("Hello from sandboxed Python!")
print("Current working directory:", __import__('os').getcwd())
print("Python version:", __import__('sys').version)

# Test basic computation
numbers = [1, 2, 3, 4, 5]
result = sum(x * x for x in numbers)
print(f"Sum of squares: {result}")

# Test file operations within sandbox
with open('output.txt', 'w') as f:
    f.write(f"Generated output: {result}\n")
    f.write("This file was created by Python script\n")

print("✅ Python script executed successfully!")

File: sandbox_workspace/src/hello.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    printf("Hello from sandboxed C program!\n");
    printf("Testing basic operations...\n");
    
    int sum = 0;
    for (int i = 1; i <= 10; i++) {
        sum += i;
    }
    
    printf("Sum of 1-10: %d\n", sum);
    
    // Create output file
    FILE *fp = fopen("c_output.txt", "w");
    if (fp != NULL) {
        fprintf(fp, "C program output: %d\n", sum);
        fclose(fp);
        printf("Output file created successfully\n");
    }
    
    return 0;
}
